healpy/projaxes.py

```diff
@@ -612,19 +595,18 @@ def graticule(
         return dpar, dmer
 
     def delgraticules(self):
-        """Delete all graticules previously created on the Axes.
-        """
+        """Delete all graticules previously created on the Axes."""
         if hasattr(self, "_graticules"):
             for dum1, dum2, g in self._graticules:
                 for gl in g:
                     for l in gl:
                         if l in self.lines:
                             self.lines.remove(l)
                         else:
-                            warnings.warn("line not in lines")
+                            log.warning("line not in lines")
```

Comment:
Is this going to spew a large amount of log chatter, since it's inside a triply nested loop?